langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -103,6 +145,10 @@ public isolated function 'join(string separator, string... strs) returns string
 } external;
 
 # Finds the first occurrence of one string in another string.
+# ```ballerina
+# string country = "New Zealand";
+# int? firstIndex = country.indexOf("land");
```

Comment:
Need to add an example with `startIndex` also.

Reply:
Applicable to some other functions also.

Reply:
Added for possible cases.